browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.js

```diff
@@ -101,5 +102,10 @@ Polymer({
   openWebStoreUrl_: function() {
     window.open(loadTimeData.getString('getMoreExtensionsUrl'));
   },
+
+  shouldShowRestartForGoogleLogin_: function(value) {
+    return this.browserProxy_.wasSignInEnabledAtStartup() != value;
```

Comment:
nit: this doesn't need to be either a function (since it's static) or something from browserProxy (since this component can access `loadTimeData` just fine). Can simply be a polymer property initialized with the value of `loadTimeData.getBoolean('signInAllowedOnNextStartupInitialValue')`. Still, just a nit and only mentioning to reduce code or for the future.